src/doc/trpl/comments.md

```diff
@@ -1,47 +1,45 @@
 % Comments
 
-Now that we have some functions, it's a good idea to learn about comments.
+Now that we have some functions, it’s a good idea to learn about comments.
```

Comment:
Out of curiosity, why the switch to a unicode apostrophe? I'm not sure that other contributors to the docs may follow that style if it's harder to type, but I dunno if it makes much of an inconsistency.